### PR TITLE
Tidy up

### PR DESCRIPTION
**Pull request by iteles**
A few tiny updates to tidy things up: 
+ Updates readme with latest status
+ Make a couple of niggly updates to the 'our process' section
+ Adds all relevant information to the footer of the site as discussed in #146 
+ Adds a short note of support on the Chagossians - we can iterate on this based on questions https://github.com/dwyl/dwyl.github.io/issues/146

@nelsonic Please review/merge. Thanks!

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles as discussed, Co + VAT number should be on the site...

**Comment by iteles** [comment]
Will add this tomorrow!

**Comment by nelsonic** [comment]
@iteles 🔔

**Comment by iteles** [comment]
Life happened, I'm on it.

File: views/index.html [hunk 1]
```diff
@@ -188,7 +187,12 @@ <h2 class="call-to-action">Let's Talk!</h2>
 
   <!-- This needs an additional class of dark-bg when 'What our clients say' gets added -->
   <div class="copyright-block">
-    Copyright 2016 &copy; dwyl Ltd. All rights reserved. 104 Mary Datchelor Close, London SE5 7DY, UK
+    <p>
+      Copyright 2016 &copy; dwyl ltd, company number 09525434, registered in England &amp; based at 104 Mary Datchelor Close, London SE5 7DY, UK. All rights reserved.
```
Comment (nelsonic) [inline views/index.html]: Thought the point was we have to list the Company & VAT number but don't *have* to publicise the address...?

Comment (iteles) [inline views/index.html]: According to [this link you sent me](http://www.out-law.com/page-7594), the registered address *Also* has to be part of the displayed information 😞

Comment (nelsonic) [inline views/index.html]: Please add a `VA-Task` to re-investigate registered offices with mail scanning services. or we could set it up at 16 Palmers Road ...

**Pull request merged**